waspc/packages/ts-inspect/package.json

```diff
@@ -11,6 +11,10 @@
     "start": "node ./dist/index.js",
     "test": "jest"
   },
+  "//": [
+    "typescript is purposefully a normal dependency, and not a dev dependency:",
+    "the compiler is called at runtime to extract information from JS/TS files."
+  ],
```

Comment:
This is apparently how people usually add comments to package.json files, since they don't support actual comments.